sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+# must be unique in a given SonarQube instance
+sonar.projectKey=com.cognifide.qa.bb
+# this is the name and version displayed in the SonarQube UI. Was mandatory prior to SonarQube 6.1.
+sonar.projectName=Bobcat
+sonar.projectVersion=1.4.1-SNAPSHOT

```

Comment:
Do we have to provide it here? How are we going to update it during a release and after one?